tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/RabbitMQ/BasicGetIntegration.cs

```diff
@@ -103,7 +105,7 @@ internal static CallTargetState OnMethodBegin<TTarget>(TTarget instance, string
                         scope.Span.SetException(exception);
                     }
 
-                    if (basicProperties != null)
+                    if (basicProperties != null && tags is not null)
```

Comment:
Same I would have done this null check inside SetDataStreamsCheckpointOnConsume. It seems more logical for this decision to be taken by the DSM code. It could be ok not to have edge tags (I assume not but you see what I mean)

Reply:
IMO, the signature of `SetDataStreamsCheckpointOnConsume` _dictates_ that's it's not ok to have edge tags. I get what you mean, but seeing as the null check is pointless anyway, I'm inclined to leave it as-is.